### PR TITLE
chore(cli): bash autocomplete for apic

### PR DESCRIPTION
**Pull request by millotp**
## 🧭 What and Why

Bringing our cli to the next level with bash autocomplete ! We could event remove `inquirer` now.

Screenshot 2023-12-27 at 16 42 18

### CODE REVIEW

File: scripts/install.sh [hunk 1]
```diff
@@ -4,7 +4,92 @@
...
 # [[ -s "$HOME/<path to api-clients-automation>/scripts/install.sh" ]] && source "$HOME/<path to api-clients-automation>/scripts/install.sh"
 
+ROOT="$(cd "$(dirname "${BASH_SOURCE[0]}")" >/dev/null && pwd)"
+
 export apic() {
   docker exec -it api-clients-automation bash -lc "cd scripts && NODE_NO_WARNINGS=1 node dist/scripts/cli/index.js $*";
 }
 
+_list_languages() {
+  cat $ROOT/config/clients.config.json | jq -r 'keys[]'
```
Comment (shortcuts) [inline scripts/install.sh:14]: Maybe it should be executed in docker so users that don't have jq (idk if that exists) doesn't have to install it

**Review by shortcuts — COMMENTED** [review]
Thats so nice!!!!!!

File: scripts/install.sh (hunk 1 above)
Comment (millotp) [reply scripts/install.sh:14]: I'm not sure if it's possible, autocomplete has to execute really fast to feel natural, and this is called everytime someone presses tab.
Maybe I can make it conditional, is jq is not installed I fallback to docker ?

Comment (shortcuts) [reply scripts/install.sh:14]: That make sense, lets not bother then, jq is required anyway elsewhere

**Review by shortcuts — APPROVED** [review]
Ggg

**Pull request merged**